### PR TITLE
add(main): new package opencolorio

### PR DESCRIPTION
**Pull request by T-Dynamos**
#11674

### CODE REVIEW

**Comment by T-Dynamos** [comment]
```
[413/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/Displays.cpp.o
[414/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/RedCameras.cpp.o
[415/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/DisplayViewTransform.cpp.o
[416/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/BuiltinTransform.cpp.o
[417/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ColorSpaceTransform.cpp.o
[418/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/CDLTransform.cpp.o
[419/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExponentTransform.cpp.o
[420/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExponentWithLinearTransform.cpp.o
[421/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExposureContrastTransform.cpp.o
[422/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/FixedFunctionTransform.cpp.o
[423/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingPrimaryTransform.cpp.o
[424/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingRGBCurveTransform.cpp.o
[425/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/FileTransform.cpp.o
[426/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingToneTransform.cpp.o
[427/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogAffineTransform.cpp.o
[428/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GroupTransform.cpp.o
[429/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogCameraTransform.cpp.o
[430/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogTransform.cpp.o
[431/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LookTransform.cpp.o
[432/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/Lut1DTransform.cpp.o
[433/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/Lut3DTransform.cpp.o
[434/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/MatrixTransform.cpp.o
[435/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/RangeTransform.cpp.o
[436/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ViewTransform.cpp.o
[437/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ViewingRules.cpp.o
[438/448] Building CXX object src/OpenColorIO/CMakeFiles/OpenColorIO.dir/SystemMonitor.cpp.o
[439/448] Linking CXX shared library src/OpenColorIO/libOpenColorIO.so
FAILED: src/OpenColorIO/libOpenColorIO.so 
: && /home/builder/.termux-build/_cache/android-r25-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r25-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r25-api-24-v1/sysroot -fPIC -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG  -L/data/data/com.termux/files/usr/lib -Wl,-rpath=/data/data/com.termux/files/usr/lib -fopenmp -static-openmp -Wl,--enable-new-dtags -Wl,--as-needed -Wl,-z,relro,-z,now --target=aarch64-linux-android24  -Wl,--exclude-libs,ALL -shared -Wl,-soname,libOpenColorIO.so -o src/OpenColorIO/libOpenColorIO.so src/OpenColorIO/CMakeFiles/OpenColorIO.dir/apphelpers/CategoryHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/apphelpers/ColorSpaceHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/apphelpers/DisplayViewHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/apphelpers/LegacyViewingPipeline.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/apphelpers/MixingHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Baker.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/BitDepthUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Caching.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ColorSpace.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ColorSpaceSet.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Config.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Context.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ContextVariableUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/CPUProcessor.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Display.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/DynamicProperty.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Exception.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/cdl/CDLParser.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/cdl/CDLReaderHelper.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/cdl/CDLWriter.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/ctf/CTFReaderHelper.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/ctf/CTFReaderUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/ctf/CTFTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/ctf/IndexMapping.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormat3DL.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatCCC.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatCC.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatCDL.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatCSP.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatCTF.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatDiscreet1DL.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatHDL.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatICC.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatIridasCube.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatIridasItx.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatIridasLook.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatPandora.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatResolveCube.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatSpi1D.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatSpi3D.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatSpiMtx.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatTruelight.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FileFormatVF.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/FormatMetadata.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/xmlutils/XMLReaderHelper.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/xmlutils/XMLReaderUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/fileformats/xmlutils/XMLWriterUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/FileRules.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/GPUProcessor.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/GpuShader.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/GpuShaderDesc.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/GpuShaderClassWrapper.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/GpuShaderUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/HashUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ImageDesc.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ImagePacking.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Logging.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Look.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/LookParse.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/MathUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/md5/md5.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/NamedTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/OCIOYaml.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Op.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/OpOptimizers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/allocation/AllocationOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/cdl/CDLOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/cdl/CDLOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/cdl/CDLOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/cdl/CDLOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/exponent/ExponentOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/exposurecontrast/ExposureContrastOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/exposurecontrast/ExposureContrastOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/exposurecontrast/ExposureContrastOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/exposurecontrast/ExposureContrastOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/fixedfunction/FixedFunctionOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/fixedfunction/FixedFunctionOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/fixedfunction/FixedFunctionOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/fixedfunction/FixedFunctionOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gamma/GammaOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gamma/GammaOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gamma/GammaOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gamma/GammaOpUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gamma/GammaOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingprimary/GradingPrimary.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingprimary/GradingPrimaryOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingprimary/GradingPrimaryOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingprimary/GradingPrimaryOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingprimary/GradingPrimaryOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingBSplineCurve.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingRGBCurveOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingRGBCurveOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingRGBCurveOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingRGBCurveOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingrgbcurve/GradingRGBCurve.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingtone/GradingTone.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingtone/GradingToneOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingtone/GradingToneOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingtone/GradingToneOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/gradingtone/GradingToneOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/log/LogOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/log/LogOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/log/LogOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/log/LogOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/log/LogUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut1d/Lut1DOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut1d/Lut1DOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut1d/Lut1DOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut1d/Lut1DOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut3d/Lut3DOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut3d/Lut3DOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut3d/Lut3DOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/lut3d/Lut3DOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/matrix/MatrixOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/matrix/MatrixOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/matrix/MatrixOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/matrix/MatrixOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/noop/NoOps.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/OpTools.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/range/RangeOpCPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/range/RangeOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/range/RangeOpGPU.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/range/RangeOp.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ops/reference/ReferenceOpData.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ParseUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/PathUtils.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Platform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Processor.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ScanlineHelper.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/Transform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/AllocationTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/ACES.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/BuiltinTransformRegistry.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/ColorMatrixHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/OpHelpers.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/ArriCameras.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/CanonCameras.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/Displays.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/PanasonicCameras.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/RedCameras.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/builtins/SonyCameras.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/BuiltinTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/CDLTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ColorSpaceTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/DisplayViewTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExponentTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExponentWithLinearTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/ExposureContrastTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/FileTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/FixedFunctionTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingPrimaryTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingRGBCurveTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GradingToneTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/GroupTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogAffineTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogCameraTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LogTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/LookTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/Lut1DTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/Lut3DTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/MatrixTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/transforms/RangeTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ViewingRules.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/ViewTransform.cpp.o src/OpenColorIO/CMakeFiles/OpenColorIO.dir/SystemMonitor.cpp.o  /data/data/com.termux/files/usr/lib/libexpat.so.1.8.8  /data/data/com.termux/files/usr/lib/libImath-3_1.so  ext/dist//data/data/com.termux/files/usr/lib/libpystring.a  /data/data/com.termux/files/usr/lib/libyaml-cpp.so  -lm && :
clang-14: error: no such file or directory: 'ext/dist//data/data/com.termux/files/usr/lib/libpystring.a'
```
PLease review request merge #11711

**Comment by Grimler91** [comment]
Seems this was closed since it was linked to libexpat fix PR

**Comment by T-Dynamos** [comment]
```
 /home/builder/.termux-build/_cache/android-r25-api-24-v1/bin/clang++ --target=aarch64-none-linux-android --gcc-toolchain=/home/builder/.termux-build/_cache/android-r25-api-24-v1 --sysroot=/home/builder/.termux-build/_cache/android-r25-api-24-v1/sysroot -DOCIO_SOURCE_DIR=/home/builder/.termux-build/opencolorio/src -DOCIO_UNIT_TEST_FILES_DIR=/home/builder/.termux-build/opencolorio/build/testdata -DOpenColorIO_SKIP_IMPORTS -I/home/builder/.termux-build/opencolorio/src/tests/cpu -I/home/builder/.termux-build/opencolorio/src/include/OpenColorIO/.. -I/home/builder/.termux-build/opencolorio/src/include/OpenColorIO -I/home/builder/.termux-build/opencolorio/src/src/OpenColorIO -I/home/builder/.termux-build/opencolorio/build/include/OpenColorIO -I/home/builder/.termux-build/opencolorio/src/tests/testutils/.. -isystem /home/builder/.termux-build/opencolorio/build/src/utils/.. -isystem /data/data/com.termux/files/usr/include/Imath -isystem /data/data/com.termux/files/usr/lib -isystem /home/builder/.termux-build/opencolorio/src/ext/sampleicc/src/include -isystem /home/builder/.termux-build/opencolorio/src/src/utils/.. -fstack-protector-strong -Oz --target=aarch64-linux-android24  -I/data/data/com.termux/files/usr/include -O3 -DNDEBUG -fPIE -fvisibility=hidden -fvisibility-inlines-hidden  -DUSE_CLANG -Wno-deprecated-register -Wall -Wextra -Wswitch-enum -std=c++11 -MD -MT tests/cpu/CMakeFiles/test_cpu_exec.dir/__/__/src/OpenColorIO/OCIOYaml.cpp.o -MF tests/cpu/CMakeFiles/test_cpu_exec.dir/__/__/src/OpenColorIO/OCIOYaml.cpp.o.d -o tests/cpu/CMakeFiles/test_cpu_exec.dir/__/__/src/OpenColorIO/OCIOYaml.cpp.o -c /home/builder/.termux-build/opencolorio/src/src/OpenColorIO/OCIOYaml.cpp
/home/builder/.termux-build/opencolorio/src/src/OpenColorIO/OCIOYaml.cpp:22:10: fatal error: 'pystring/pystring.h' file not found
#include "pystring/pystring.h"
         ^~~~~~~~~~~~~~~~~~~~~
1 error generated.
[30/440] Building CXX object tests/cpu/CMakeFiles/test_cpu_exec.dir/__/__/src/OpenColorIO/ops/cdl/CDLOpGPU.cpp.o
```
Why still not fixed by #11732

**Comment by Grimler91** [comment]
Probably `pystring.h` should have been installed to `$PREFIX/include/pystring/` instead of `$PREFIX/include/`, since it looks for `"pystring/pystring.h"`

**Comment by T-Dynamos** [comment]
@Grimler91 Please open this pr

**Comment by T-Dynamos** [comment]
```
-- Found PythonInterp: /usr/bin/python3.10 (found version "3.10.4") 
CMake Error at tools/FindPythonLibsNew.cmake:161 (message):
  Python config failure: Python is 64-bit, chosen compiler is 32-bit
Call Stack (most recent call first):
  tools/pybind11Tools.cmake:38 (find_package)
  tools/pybind11Common.cmake:201 (include)
  CMakeLists.txt:169 (include)


-- Configuring incomplete, errors occurred!
See also "/home/builder/.termux-build/opencolorio/build/ext/build/pybind11/src/pybind11_install-build/CMakeFiles/CMakeOutput.log".
[8/440] Building CXX object tests/utils/CMakeFiles/test_utils_exec.dir/StringUtils_tests.cpp.o
ninja: build stopped: subcommand failed.
```
How to fix it?

**Comment by ifurther** [comment]
Please remove link libexpat.so.1.8.8, and it should binding Python?

**Comment by ifurther** [comment]
Maybe close python binding to build, set `-DOCIO_BUILD_PYTHON=off`

**Comment by T-Dynamos** [comment]
Can anybody merge this as required for #12103

File: packages/opencolorio/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://opencolorio.org
+TERMUX_PKG_DESCRIPTION="A color management framework for visual effects and animation"
+TERMUX_PKG_LICENSE="BSD"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=2.1.2
+TERMUX_PKG_SRCURL=https://github.com/AcademySoftwareFoundation/OpenColorIO/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6c6d153470a7dbe56136073e7abea42fa34d06edc519ffc0a159daf9f9962b0b
+TERMUX_PKG_DEPENDS="freeglut, glew, littlecms, libtinyxml2, libyaml-cpp, imath, libexpat, pystring"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-Dpystring_INCLUDE_DIR=/data/data/com.termux/files/usr/lib/ -DOCIO_BUILD_PYTHON=off"
```
Comment (Grimler91) [inline packages/opencolorio/build.sh:9]: Use $TERMUX_PREFIX instead of /data/data/com.termux/files/usr, we do not want to hardcode the prefix string

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Comment by T-Dynamos** [comment]
Not merged yet?

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**